scripts/schemaupgrade.sql

```diff
@@ -1,6 +1,28 @@
-alter table channels change column owner_userid owner_user_id int;
-alter table channels change column topic_userid topic_user_id int;
-alter table channels drop owner;
-alter table channels drop topic_owner;
+-- #40 renames.new is redundant
+alter table renames drop new;
 
+-- #61 rename lobby_id in logins to agent
+alter table logins change column lobby_id agent;
 
+-- #333 cleanup channel
+-- ???
+
+-- #359 uniqueness for dbuser.email
+alter table user add unique(email);
+
+-- #360 split last_id
```

Comment:
is anything hurt when the the existing values in logins aren't updated? i'll just clear the table for now. dropping the unused rows should be done afterwards when the updated was a success.

Reply:
... as its truncated the unused rows can be removed instantly

Reply:
the existing values could stay, but since the table is currently not used anywhere (afaik) i think the simplest thing is to simply wipe the logins table clean